packages/@jsii/dotnet-analyzers/CHANGELOG.md

```diff
@@ -1,214 +0,0 @@
-
-
-
-
-
-# [0.22.0](https://github.com/aws/jsii/compare/v0.21.2...v0.22.0) (2020-02-06)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.21.2](https://github.com/aws/jsii/compare/v0.21.1...v0.21.2) (2020-01-22)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.21.1](https://github.com/aws/jsii/compare/v0.21.0...v0.21.1) (2020-01-03)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-# [0.21.0](https://github.com/aws/jsii/compare/v0.20.11...v0.21.0) (2020-01-02)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.20.11](https://github.com/aws/jsii/compare/v0.20.10...v0.20.11) (2019-12-13)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.20.10](https://github.com/aws/jsii/compare/v0.20.9...v0.20.10) (2019-12-13)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.20.9](https://github.com/aws/jsii/compare/v0.20.8...v0.20.9) (2019-12-11)
-
-**Note:** Version bump only for package @jsii/dotnet-analyzers
-
-
-
-
-
-## [0.20.8](https://github.com/aws/jsii/compare/v0.20.7...v0.20.8) (2019-11-24)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.7](https://github.com/aws/jsii/compare/v0.20.5...v0.20.7) (2019-11-18)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.6](https://github.com/aws/jsii/compare/v0.20.5...v0.20.6) (2019-11-14)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.5](https://github.com/aws/jsii/compare/v0.20.4...v0.20.5) (2019-11-13)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.4](https://github.com/aws/jsii/compare/v0.20.3...v0.20.4) (2019-11-12)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.3](https://github.com/aws/jsii/compare/v0.20.2...v0.20.3) (2019-11-11)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.2](https://github.com/aws/jsii/compare/v0.20.1...v0.20.2) (2019-11-08)
-
-**Note:** Version bump only for package jsii-dotnet-analyzers
-
-
-
-
-
-## [0.20.1](https://github.com/aws/jsii/compare/v0.20.0...v0.20.1) (2019-11-06)
-
-
-### Bug Fixes
-
-* **dotnet/analyzer:** remove dependency on Runtime ([#927](https://github.com/aws/jsii/issues/927)) ([815b449](https://github.com/aws/jsii/commit/815b44982bfe1f9b2ee3a9cf60e4f5dfb4dd22f6))
-
-
-
-
-
-## [0.20.0](https://github.com/aws/jsii/compare/v0.19.0...v0.20.0) (2019-10-30)
-
-
-### Bug Fixes
-
-* **java,dotnet:** emit default implementations for optional properties ([#906](https://github.com/aws/jsii/issues/906)) ([37ddfd5](https://github.com/aws/jsii/commit/37ddfd5fde1399274ca1541542c7268b75e026c2)), closes [#543](https://github.com/aws/jsii/issues/543)
-
```

Comment:
Should this get merged in with the changelog in dotnet-runtime ?

I'll leave this to you since I don't know how these changelogs are used.

Reply:
Good point. I'd rather get rid of per-package changelogs in the long run... so I'l just accept this as civilian causalties.